.gitignore

```diff
@@ -10,6 +10,7 @@ tmp
 .cache-loader
 .attest
 coverage
+/**/themes/*.json
```

Comment:
Ideally in the future this should be the specific path to the folder you want to exclude, since it's not like this is being generated anywhere else